wp-content/themes/core/components/header/subheader_archive/css/subheader_arcihve.pcss

```diff
@@ -0,0 +1,18 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Component: Subheader Archive
+ *
+ * ----------------------------------------------------------------------------- */
+
+.c-subheader-archive {
+	.c-subheader--has-image & {
+	    align-items: center;
+		padding: var(--spacer-20) 0 var(--spacer-20) 0;
+		min-height: 580px;
+	}
+
+	.c-subheader__description {
+		width: calc(75% - var(--grid-gutter));
```

Comment:
Please use the existing grid width classes for this rather than a 1-off width.

Reply:
This code is gone now.